server/controllers/overviewController.ts

```diff
@@ -53,17 +54,20 @@ export default class OverviewController {
         prisonerData,
         staffId: res.locals.user.staffId,
         inmateDetail,
-        apiErrorCallback: res.locals.apiErrorCallback,
+        apiErrorCallback,
         userCaseLoads: res.locals.user.caseLoads,
         userRoles: res.locals.user.userRoles,
       }),
       pathfinderApiClient.getNominal(prisonerData.prisonerNumber),
       manageSocCasesApiClient.getNominal(prisonerData.prisonerNumber),
       this.offencesService.getNextCourtHearingSummary(clientToken, prisonerData.bookingId),
       this.offencesService.getActiveCourtCasesCount(clientToken, prisonerData.bookingId),
-      showCourtCaseSummary
-        ? this.offencesService.getLatestReleaseCalculation(clientToken, prisonerData.prisonerNumber)
-        : null,
+      Result.wrap(
+        showCourtCaseSummary
+          ? this.offencesService.getLatestReleaseCalculation(clientToken, prisonerData.prisonerNumber)
+          : null,
+        apiErrorCallback,
```

Comment:
By providing the apiErrorCallback, you get the error banner appearing at the top of the page